docs/installation/ubuntu.md

```diff
@@ -0,0 +1,162 @@
+### Ubuntu 18.04+ 
+
+This set of instruction is meant to install of AutoLab v2.40 on Ubuntu 18.04 LTS.
```

Comment:
Update it to say this set of instruction is meant to install Autolab on Ubuntu 18.04 LTS

Reply:
Updated